spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -183,6 +187,21 @@
 				<artifactId>google-cloud-logging-logback</artifactId>
 				<version>0.32.0-alpha</version>
 			</dependency>
+
+			<dependency>
+				<groupId>org.springframework.integration</groupId>
+				<artifactId>spring-integration-core</artifactId>
+				<version>${spring-integration.version}</version>
+			</dependency>
+
+			<!-- google-cloud-bom is setting the version of netty-tcnative-boringssl-static which is incompatible
+			with SB2.
+			Remove when https://github.com/GoogleCloudPlatform/google-cloud-java/issues/2781 is resolved. -->
+			<dependency>
+				<groupId>io.netty</groupId>
+				<artifactId>netty-tcnative-boringssl-static</artifactId>
+				<version>2.0.7.Final</version>
```

Comment:
This is likely incompatible with grpc 1.7 & 1.8

https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty

Reply:
I'm able to run my test app (which uses Tomcat) and publish/subscribe from Google Cloud Pub/Sub.